src/util/maputil/maputil.go

```diff
@@ -0,0 +1,13 @@
+// Package maputil provides Go maps related utility methods
+package maputil
+
+// Copy copies the map contents to the new map
+func Copy(data map[string]string) map[string]string {
```

Comment:
I would keep this in package kvstorage since it is specific to map[string]string